bugsnag-plugin-react-native/src/main/java/com/bugsnag/android/BugsnagReactNativePlugin.kt

```diff
@@ -0,0 +1,104 @@
+package com.bugsnag.android
+
+import java.util.Locale
+
+@Suppress("unused", "UNUSED_PARAMETER")
+class BugsnagReactNativePlugin : Plugin {
+
+    private val configSerializer = ConfigSerializer()
+    private val appSerializer = AppSerializer()
+    private val deviceSerializer = DeviceSerializer()
+    private val breadcrumbSerializer = BreadcrumbSerializer()
+    private val threadSerializer = ThreadSerializer()
+
+    internal lateinit var internalHooks: InternalHooks
+    internal lateinit var client: Client
+
+    lateinit var logger: Logger
+
+    override fun load(client: Client) {
+        client.logger.i("Initialized React Native Plugin")
+    }
+
+    override fun unload() {}
+
+    fun configure(): Map<String, Any?> {
+        try {
+            // see if bugsnag-android is already initialised
+            client = Bugsnag.getClient()
+            internalHooks = InternalHooks(client)
+            logger = client.logger
+        } catch (exc: IllegalStateException) {
+            throw exc
+        }
```

Comment:
This doesn't appear to do anything.

Reply:
This is part of the previous implementation in the bugsnag-js repo. I agree that it's redundant but I left it unchanged for now to make the changeset easier to review.